### PR TITLE
💬 TOC.md supports xml comment

### PR DESCRIPTION
**Pull request by yufeih**
Fixes https://ceapex.visualstudio.com/Engineering/_git/Docs.DocFX.Impact/pullrequest/8891?_a=files&path=%2Fsource%2Fazure-docs-pr.ja-jp.output%2Fbuild.log

### CODE REVIEW

File: src/docfx/build/toc/parser/MarkdownTocMarkup.cs [hunk 1]
```diff
@@ -15,24 +14,26 @@ namespace Microsoft.Docs.Build
 {
     internal static class MarkdownTocMarkup
     {
-        private static readonly HashSet<Type> s_blockWhiteList = new HashSet<Type> { typeof(HeadingBlock) /*header*/, typeof(YamlFrontMatterBlock) /*yaml header*/, typeof(HtmlBlock) /*comment*/ };
```
Comment (OsmondJiang) [inline src/docfx/build/toc/parser/MarkdownTocMarkup.cs]: why need change that? we should only need add html block parser back?

Comment (yufeih) [reply src/docfx/build/toc/parser/MarkdownTocMarkup.cs]: There are other types of html block types, we only support comments.

File: src/docfx/build/toc/parser/MarkdownTocMarkup.cs [hunk 2]
```diff
@@ -4,7 +4,6 @@
 using System;
 using System.Collections.Generic;
 using System.Diagnostics;
-using System.Linq;
```
Comment (live1206) [inline src/docfx/build/toc/parser/MarkdownTocMarkup.cs]: seems like this is still needed?

Comment (yufeih) [reply src/docfx/build/toc/parser/MarkdownTocMarkup.cs]: yes, introduced by accident

**Pull request merged**